contrib/mkimage-archarm-pacman.conf

```diff
@@ -0,0 +1,98 @@
+#
+# /etc/pacman.conf
+#
+# See the pacman.conf(5) manpage for option and repository directives
+
+#
+# GENERAL OPTIONS
+#
+[options]
+# The following paths are commented out with their default values listed.
+# If you wish to use different paths, uncomment and update the paths.
+#RootDir     = /
+#DBPath      = /var/lib/pacman/
+#CacheDir    = /var/cache/pacman/pkg/
+#LogFile     = /var/log/pacman.log
+#GPGDir      = /etc/pacman.d/gnupg/
+HoldPkg     = pacman glibc
+#XferCommand = /usr/bin/curl -C - -f %u > %o
+#XferCommand = /usr/bin/wget --passive-ftp -c -O %o %u
+#CleanMethod = KeepInstalled
+#UseDelta    = 0.7
+Architecture = armv7h
```

Comment:
Ah nice -- I'm personally running Docker successfully on ARMv5 with Debian,
and I know lots of folks are doing so successfully on ARMv6 (especially the
RPi variant), so I'm +1 for some kind of parameterization.  If it's just
tweaking this one field, it might be easiest to replace it with something
like "%ARCH%" in the file here and just "sed" it appropriately in the
script; WDYT?